internal/eval/store_eval.go

```diff
@@ -600,3 +600,258 @@ func evalPFMERGE(args []string, store *dstore.Store) *EvalResponse {
 		Error:  nil,
 	}
 }
+
+// helper function to insert key value in hashmap associated with the given hash
+func insertInHashMap(args []string, store *dstore.Store) (int64, error) {
```

Comment:
In this full function, I see at one place we are sending dicerrors, at other sending normal errors. Let's keep it consistent and send dicerrors from the helper function and return back directly from the respective eval.